src/frontend-pwa/src/components/common/Header/Header.tsx

```diff
@@ -1,6 +1,6 @@
 /**
  * @summary Reusable BC Gov Header Component
- * @author Dallas Richmond
+ * @author Dallas Richmond, Matthew Logan
```

Comment:
```suggestion
 * @author Dallas Richmond, LocalNewsTV
```
Just keeping it consistent